foia_hub/scripts/load_agency_contacts.py

```diff
@@ -8,10 +8,10 @@
 import yaml
 from glob import iglob
 import django
-django.setup()
 
 from foia_hub.models import Agency, Office, Stats, ReadingRoomUrls
 
+django.setup()
```

Comment:
This change is related to the fact that the new flake8 likes all the imports up top.